### PR TITLE
feat: gateway mvp - upload and download single files

### LINKED ISSUE #17: download: info page before downloading
- [ ] minimum is a blank page with a download button
- display metadata about the content to be downloaded
    - [ ] size?
    - [ ] filename?
    - [ ] file type?
    - [ ] custom message/note by uploader?
    - [ ] could be just in the URL itself?

### PR DESCRIPTION
**Pull request by vojtechsimetka**
Resolves #17, #11, #7, #6, #1

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@500&display=swap" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@300&display=swap" rel="stylesheet" />
```
Comment (vojtechsimetka) [inline public/index.html:27]: Should be directly included instead of using googleapis

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Bee Dashboard",
+  "name": "Bee Dashboard",
```
Comment (Cafe137) [inline public/manifest.json:3]: Gateway

**Review by nugaon — COMMENTED** [review]
.gitignore file is missing, please don't forget to add:
- node_modules
- build

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta name="description" content="Web site created using create-react-app" />
```
[the remark this answers is not in the document]
Comment (agazso) [reply public/index.html:8]:
```suggestion
    <meta name="description" content="The easiest way to share & access files on the Swarm network." />
```

**Review by agazso — APPROVED** [review]

File: public/manifest.json (hunk 2 above)
Comment (vojtechsimetka) [reply public/manifest.json:3]:
```suggestion
  "short_name": "Gateway",
  "name": "Gateway",
```

File: readme.md [hunk 4]
```diff
@@ -1,7 +1,53 @@
...
+npm start
+```
+
+The Gateway runs in development mode on [http://localhost:3030/](http://localhost:3030/). 
+
+:warn: Don't forget to provide postage stamp via environment variable or edit directly the `postageStamp` constant in [`.src/constants.ts`](./src/constants.ts)..
+
+### Environment variables
+- `REACT_APP_BEE_API` - Bee API URL (defaults to `http://localhost:1633`)
+- `REACT_APP_POSTAGE_STAMP` - Postage stamp batch ID to be used for uploading
```
[the remark this answers is not in the document]
Comment (vojtechsimetka) [reply readme.md:36]:
```suggestion
### Environment variables
- `REACT_APP_BEE_DOWNLOAD_API` - Bee API URL from which the gateway downloads (defaults to `http://localhost:1633`)
- `REACT_APP_BEE_UPLOAD_APIS` - comma separated bee API URLs through which the gateway uploads. The api to upload is selected at random (defaults to `[http://localhost:1633]`)
- `REACT_APP_POSTAGE_STAMP` - Postage stamp batch ID to be used for uploading (defaults to `00000...00000`
- `REACT_APP_GATEWAY_URL` - URL on which the gateway is hosted (defaults to current window location)
```

**Pull request merged**